src/Common/QueryFuzzer.cpp

```diff
@@ -433,69 +484,60 @@ void QueryFuzzer::fuzzWindowFrame(ASTWindowDefinition & def)
 {
     switch (fuzz_rand() % 40)
     {
-        case 0:
-        {
+        case 0: {
```

Comment:
Minor nitpick: Why did you move the start of the curly brace one line?
I think according to our coding style it should be on a separate line as it was before

Reply:
I used clang-format and it didn't format this? :thinking:

Reply:
I used clang-format in the whole file and it formated this. Am I doing something wrong?

Reply:
Probably not. Maybe it's a rule that is not set up properly in our clang-format file. It's not a big thing anyway
[Point 3 of our C++ coding style](https://clickhouse.com/docs/development/style#formatting)

Point 1 says:
> 1. Most of the formatting is done automatically by clang-format.

This is not one of those I guess :sweat: